benchmarks/src/main/java/io/druid/benchmark/ColumnScanBenchmark.java

```diff
@@ -0,0 +1,292 @@
+package io.druid.benchmark;
```

Comment:
i wonder fi we should document the list of benchmarks somewhere

will make it easy for developers to test their work